### PR TITLE
Cal 202 add supported file types to documentation

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Adds DDF & Alliance supported file types to documentation.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?

@peterhuffer @lcrosenbu @troymohl 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@pklinef
@shaundmorris

#### How should this be tested?
verify content
#### Any background context you want to provide?

Depends on [DDF-2603](https://codice.atlassian.net/browse/DDF-2603), see: https://github.com/codice/ddf/pull/1449
#### What are the relevant tickets?

[CAL-202](https://codice.atlassian.net/browse/CAL-202)
[DDF-2603](https://codice.atlassian.net/browse/DDF-2603)
#### Checklist:
- [x] Documentation Updated
	- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+* `ext.nitf.csexra.num-lines`
+* `ext.nitf.csexra.num-samples`
+* `ext.nitf.csexra.angle-to-north`
+* `ext.nitf.csexra.obliquity-angle`
+* `ext.nitf.csexra.azimuth-obliquity`
+* `ext.nitf.csexra.sun-azimuth`
+* `ext.nitf.csexra.sun-elevation`
+* `ext.nitf.csexra.circular-error`
+* `ext.nitf.csexra.linear-error`
+* `sr.cloud-cover`
```
Comment (peterhuffer) [inline distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc]: typo: `sr.cloud-cover` -> `isr.cloud-cover`

**Comment by peterhuffer** [comment]
We also support these
`
ACFTB Attributes
ext.nitf.acftb.aircraft-mission-id
ext.nitf.acftb.aircraft-tail-number
ext.nitf.acftb.sensor-id-type
ext.nitf.acftb.sensor-id
ext.nitf.acftb.aircraft-take-off
ext.nitf.acftb.scene-source
ext.nitf.acftb.scene-number
ext.nitf.acftb.processing-date
ext.nitf.acftb.immediate-scene-host
ext.nitf.acftb.immediate-scene-request-id
ext.nitf.acftb.mission-plan-mode

AIMIDB Attributes
ext.nitf.aimidb.acquisition-date
ext.nitf.aimidb.mission-number
ext.nitf.aimidb.mission-identification
ext.nitf.aimidb.flight-number
ext.nitf.aimidb.image-operation-number
ext.nitf.aimidb.current-segment
ext.nitf.aimidb.reprocess-number
ext.nitf.aimidb.replay
ext.nitf.aimidb.start-tile-column
ext.nitf.aimidb.start-tile-row
ext.nitf.aimidb.end-segment
ext.nitf.aimidb.end-tile-column
ext.nitf.aimidb.end-tile-row
ext.nitf.aimidb.country-code
ext.nitf.aimidb.location

MTIRPB Attributes
ext.nitf.mtirpb.target-location
ext.nitf.mtirpb.target-classification-category
ext.nitf.mtirpb.target-amplitude
ext.nitf.mtirpb.target-heading
ext.nitf.mtirpb.target-ground-speed
ext.nitf.mtirpb.target-radial-velocity
ext.nitf.mtirpb.target-location-accuracy
ext.nitf.mtirpb.number-of-valid-targets
ext.nitf.mtirpb.aircraft-location
ext.nitf.mtirpb.aircraft-altitude
ext.nitf.mtirpb.aircraft-altitude-unit-of-measure
ext.nitf.mtirpb.aircraft-heading
ext.nitf.mtirpb.cosine-of-graze-angle
ext.nitf.mtirpb.destination-point
ext.nitf.mtirpb.mti-left-or-right
ext.nitf.mtirpb.patch-number
ext.nitf.mtirpb.scan-date-and-time
ext.nitf.mtirpb.squint-angle
ext.nitf.mtirpb.wide-area-mti-frame-number
ext.nitf.mtirpb.wide-area-mti-bar-number
`

Attributes with an `ext.nitf.xyz` name are the original attribute name. Some of these map to a taxonomy term. Do we want both the `ext.nitf.xyz` and its mapped taxonomy attribute name if they both exist?

**Comment by clockard** [comment]
@peterhuffer are those additional attributes supported in the 0.1.x branch.

**Comment by peterhuffer** [comment]
@clockard Some of the MTIRPB and ACTFB attributes are supported in 0.1.x, but more were added to master, as well as some renaming to the attributes.

**Comment by peterhuffer** [comment]
This should be a pretty complete list of what we support in the 0.1.x branch for NITF file types. There is no taxonomy mappings in 0.1.x as far as I know. Somebody can double check this by looking at the various NITF Attribute Java files [here](https://github.com/codice/alliance/tree/0.1.x/catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf) if they wish.
```
        ACFTB Attributes
        ———————————
        nitf.acftb.aircraft-mission-id
        nitf.acftb.aircraft-tail-number
        nitf.acftb.aircraft-take-off
        nitf.acftb.sensor-id-type
        nitf.acftb.sensor-id
        nitf.acftb.scene-source
        nitf.acftb.scene-number
        nitf.acftb.processing-date
        nitf.acftb.immediate-scene-host
        nitf.acftb.immediate-scene-request-id
        nitf.acftb.mission-plan-mode

        Header Attributes
        ————————————
        nitf.file-profile-name
        nitf.file-version
        nitf.complexity-level
        nitf.standard-type
        nitf.originating-station-id
        nitf.file-date-and-time
        nitf.file-title
        nitf.file-security-classification
        nitf.file-classification-security-system
        nitf.file-codewords
        nitf.file-control-and-handling
        nitf.file-releasing-instructions
        nitf.file-declassification-type
        nitf.file-declassification-date
        nitf.file-declassification-exemption
        nitf.file-downgrade
        nitf.file-downgrade-date
        nitf.file-classification-text
        nitf.file-classification-authority-type
        nitf.file-classification-authority
        nitf.file-classification-reason
        nitf.file-security-source-date
        nitf.file-security-control-number
        nitf.file-copy-number
        nitf.file-number-of-copies
        nitf.file-background-color
        nitf.originators-name
        nitf.originators-phone-number

        MTIRPB Attributes
        ————————————
        nitf.mtirpb.target-classification-category
        nitf.mtirpb.target-amplitude
        nitf.mtirpb.target-heading
        nitf.mtirpb.target-ground-speed
        nitf.mtirpb.target-radial-velocity
        nitf.mtirpb.target-location-accuracy
        nitf.mtirpb.target-location
        nitf.mtirpb.patch-number
        nitf.mtirpb.wide-area-mti-frame-number
        nitf.mtirpb.wide-area-mti-bar-number
        nitf.mtirpb.scan-date-and-time
        nitf.mtirpb.aircraft-location
        nitf.mtirpb.aircraft-altitude
        nitf.mtirpb.aircraft-altitude-unit-of-measure
        nitf.mtirpb.aircraft-heading
        nitf.mtirpb.mti-left-or-right
        nitf.mtirpb.cosine-of-graze-angle
        nitf.mtirpb.number-of-valid-targets
        nitf.mtirpb.destination-point
        nitf.mtirpb.squint-angle

        Image Attributes
        ————————————
        nitf.image.file-part-type
        nitf.image.image-identifier-1
        nitf.image.image-date-and-time
        nitf.image.target-identifier
        nitf.image.image-identifier-2
        nitf.image.image-security-classification
        nitf.image.image-classification-security-system
        nitf.image.image-codewords
        nitf.image.image-control-and-handling
        nitf.image.image-releasing-instructions
        nitf.image.image-declassification-type
        nitf.image.image-declassification-date
        nitf.image.image-declassification-exemption
        nitf.image.image-downgrade
        nitf.image.image-downgrade-date
        nitf.image.image-classification-text
        nitf.image.image-classification-authority-type
        nitf.image.image-classification-authority
        nitf.image.image-classification-reason
        nitf.image.image-security-source-date
        nitf.image.image-security-control-number
        nitf.image.imageSource
        nitf.image.number-of-significant-rows-in-image
        nitf.image.number-of-significant-columns-in-image
        nitf.image.pixel-value-type
        nitf.image.image-representation
        nitf.image.image-category
        nitf.image.actual-bits-per-pixel-per-band
        nitf.image.pixel-justification
        nitf.image.image-coordinate-representation
        nitf.image.number-of-image-comments
        nitf.image.image-comment-1
        nitf.image.image-comment-2
        nitf.image.image-comment-3
        nitf.image.image-compression
        nitf.image.number-of-bands
        nitf.image.image-mode
        nitf.image.number-of-blocks-per-row
        nitf.image.number-of-blocks-per-column
        nitf.image.number-of-pixels-per-block-horizontal
        nitf.image.number-of-pixels-per-block-vertical
        nitf.image.number-of-bits-per-pixel
        nitf.image.image-display-level
        nitf.image.image-attachment-level
        nitf.image.image-location
        nitf.image.image-magnification

        Graphic Attributes
        ————————————
        nitf.graphic.file-part-type
        nitf.graphic.graphic-identifier
        nitf.graphic.graphic-name
        nitf.graphic.graphic-security-classification
        nitf.graphic.graphic-classification-security-system
        nitf.graphic.graphic-codewords
        nitf.graphic.graphic-control-and-handling
        nitf.graphic.graphic-releasing-instructions
        nitf.graphic.graphic-declassification-type
        nitf.graphic.graphic-declassification-date
        nitf.graphic.graphic-declassification-exemption
        nitf.graphic.graphic-downgrade
        nitf.graphic.graphic-downgrade-date
        nitf.graphic.graphic-classification-text
        nitf.graphic.graphic-classification-authority-type
        nitf.graphic.graphic-classification-authority
        nitf.graphic.graphic-classification-reason
        nitf.graphic.graphic-security-source-date
        nitf.graphic.graphic-security-control-number
        nitf.graphic.graphic-display-level
        nitf.graphic.graphic-attachment-level
        nitf.graphic.graphic-location
        nitf.graphic.graphic-color
        nitf.graphic.graphic-extended-subheader-data-length

        Label Attributes
        ————————————
        nitf.label.file-part-type
        nitf.label.label-id
        nitf.label.label-security-classification
        nitf.label.label-codewords
        nitf.label.label-control-and-handling
        nitf.label.label-releasing-instructions
        nitf.label.label-classification-authority
        nitf.label.label-security-control-number
        nitf.label.label-security-downgrade
        nitf.label.label-downgrading-event
        nitf.label.label-cell-width
        nitf.label.label-cell-height
        nitf.label.label-display-level
        nitf.label.attachment-level
        nitf.label.label-location
        nitf.label.label-text-color
        nitf.label.label-background-color
        nitf.label.extended-subheader-data-length

        SymbolAttributes
        ————————————
        nitf.symbol.file-part-type
        nitf.symbol.symbol-id
        nitf.symbol.symbol-name
        nitf.symbol.symbol-security-classification
        nitf.symbol.symbol-codewords
        nitf.symbol.symbol-control-and-handling
        nitf.symbol.symbol-releasing-instructions
        nitf.symbol.symbol-classification-authority
        nitf.symbol.symbol-security-control-number
        nitf.symbol.symbol-security-downgrade
        nitf.symbol.symbol-downgrading-event
        nitf.symbol.symbol-type
        nitf.symbol.number-of-lines-per-symbol
        nitf.symbol.number-of-pixels-per-line
        nitf.symbol.line-width
        nitf.symbol.number-of-bits-per-pixel
        nitf.symbol.display-level
        nitf.symbol.attachment-level
        nitf.symbol.symbol-location
        nitf.symbol.second-symbol-location
        nitf.symbol.symbol-color
        nitf.symbol.symbol-number
        nitf.symbol.symbol-rotation
        nitf.symbol.extended-subheader-data-length

        TextAttributes
        ————————————
        nitf.text.file-part-type
        nitf.text.text-identifier
        nitf.text.text-attachment-level
        nitf.text.text-date-and-time
        nitf.text.text-title
        nitf.text.text-security-classification
        nitf.text.text-classification-security-system
        nitf.text.text-codewords
        nitf.text.text-control-and-handling
        nitf.text.text-releasing-instructions
        nitf.text.text-declassification-type
        nitf.text.text-declassification-date
        nitf.text.text-declassification-exemption
        nitf.text.text-downgrade
        nitf.text.text-downgrade-date
        nitf.text.text-classification-text
        nitf.text.text-classification-authority-type
        nitf.text.text-classification-authority
        nitf.text.text-classification-reason
        nitf.text.text-security-source-date
        nitf.text.text-security-control-number
        nitf.text.text-format
        nitf.text.text-extended-subheader-data-length
```

File: distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc [hunk 2]
```diff
@@ -0,0 +1,86 @@
+
+${cal-branding} includes support for accessing additional metadata from specific file formats, primarily NITF. In addition to the above attributes, ${branding} can extract these attributes from NITF resources, if populated:
+
+.${cal-branding} Additional NITF Attributes
```
Comment (peterhuffer) [inline distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc]: Not sure if the taxonomy is relevant to the Alliance 0.1.x branch.

Comment (lcrosenbu) [inline distribution/docs/src/main/resources/_contents/_formats/supported-file-formats-contents.adoc]: Yes, for 0.1.x please disregard.

**Pull request merged**